ycms/static/src/types/index.d.ts

```diff
@@ -17,3 +17,5 @@ declare module "lucide/dist/esm/replaceElement" {
     const replaceElement: (element: Element, { nameAttr, icons, attrs }: ReplaceElementOptions) => void;
     export default replaceElement;
 }
+
+declare let FlowbiteInstances: any;
```

Comment:
```suggestion
declare let FlowbiteInstances;
```
As mentioned above, eslint warns us about `no-explicit-any`. I am actually a bit unsure why the CircleCI flow passed, since manually executing `eslint .` or `./tools/eslint.sh` threwo multiple warning and errors